modules/dnn/src/layers/layers_common.simd.hpp

```diff
@@ -129,38 +129,40 @@ void fastConv( const float* weights, size_t wstep, const float* bias,
 #if CV_AVX512_SKX // AVX512VL is necessary to avoid register spilling
             if (vecsize >= 32)
             {
+                __m512 r0 = _mm512_loadu_ps(rptr);
+                __m512 r1 = _mm512_loadu_ps(rptr + vecsize_aligned);
+                __m512 r2 = _mm512_loadu_ps(rptr + vecsize_aligned*2);
+                __m512 r3 = _mm512_loadu_ps(rptr + vecsize_aligned*3);
+
                 __m512 vs00_5 = _mm512_setzero_ps(), vs01_5 = _mm512_setzero_ps(),
                        vs02_5 = _mm512_setzero_ps(), vs03_5 = _mm512_setzero_ps(),
                        vs10_5 = _mm512_setzero_ps(), vs11_5 = _mm512_setzero_ps(),
                        vs12_5 = _mm512_setzero_ps(), vs13_5 = _mm512_setzero_ps(),
                        vs20_5 = _mm512_setzero_ps(), vs21_5 = _mm512_setzero_ps(),
                        vs22_5 = _mm512_setzero_ps(), vs23_5 = _mm512_setzero_ps();
 
+
                 for (; k <= vecsize - 16; k += 16, rptr += 16)
```

Comment:
> rptr += 16

Looks like this blocks such code optimization.

[Failed tests](http://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/582/steps/test_dnn/logs/stdio).